setup.py

```diff
@@ -73,6 +91,101 @@
         'export TORCH_CUDA_ARCH_LIST="compute capability" before running setup.py.\n',
     )
 
+# ----------------------------------------------------------------------
+# CUDA & nvcc compile configurations check
+
+
+def get_cuda_bare_metal_version(cuda_dir):
```

Comment:
https://github.com/EleutherAI/oslo/blob/main/oslo/torch/_C/__init__.py#L287

We already have this. I think it would be checking lazy. what do you think?